packages/geo/src/lib/import-export/shared/import.utils.ts

```diff
@@ -48,20 +50,115 @@ export function addLayerAndFeaturesToMap(features: Feature[], map: IgoMap, layer
   return layer;
 }
 
+export function addLayerAndFeaturesStyledToMap(features: Feature[], map: IgoMap, layerTitle: string,
+                                               styleListService: StyleListService, styleService: StyleService): VectorLayer {
+  const olFeatures = features.map((feature: Feature) => featureToOl(feature, map.projection));
+  let style;
+
+  if (styleListService.getStyleList(layerTitle.toString() + '.styleByAttribute')) {
+    const styleByAttribute: StyleByAttribute = {
```

Comment:
You can make a cleaner code with less repetition. Maybe with array.map, or better, get all keys in layerTitle.toString() + '.styleByAttribute' ?